src/AppBundle/Form/MembershipChooseNearbyCommitteeType.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AppBundle\Form;
+
+use AppBundle\Entity\Committee;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class MembershipChooseNearbyCommitteeType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('committees', EntityType::class, [
```

Comment:
You could remove this field it's useless, it means you can remove `buildForm` method.

Instead set `EntityType` as parent, and use `configureOptions` to define the options. In your controller you will be able to replace `$form->get('committees')->getData()` by `$form->getData()`.